styles/module/header.css

```diff
@@ -0,0 +1,85 @@
+header{
+	position: absolute;
+	top: 0;
+	width: 100%;
+	color: #ffffff;
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+	flex-wrap: wrap;
+	padding: 10px 20px;
+	transition: all 0.4s ease;
+	background: rgba(0,0,0,0.65);
+}
+
+header h1 {
+	font-size:2em;
+	font-weight: 300;
+	transition: all 0.4s ease;
+}
+header h1 a {
+	color: #ffffff;
+	text-decoration: none;
+}
+header h1 span {
+	display: inline-block;
+	background: url(../../img/logo-icon.png) no-repeat;
+	background-size: 100%;
+	width: 25px;
+	height: 25px;
+	transition: all 0.4s ease;
+}
+.mobile-menu {
+	position: absolute;
+	top:10px;
+	right: 20px;
+	color: #ffffff;
+	font-size: 2em;
+}
+header nav{
```

Comment:
update for consistency between css call and opening bracket. Choose between
.classname {
Or 
.classname{